Make-PluginMaster.ps1

```diff
@@ -52,6 +53,13 @@ Foreach-Object {
     }
     $content | add-member -Force -Name "DownloadCount" $dlCount -MemberType NoteProperty
 
+    if ($content.CategoryTags -eq $null) {
+        $fallbackCategoryTags = $categoryFallbacksMap | Select-Object -ExpandProperty $content.InternalName
+        if ($fallbackCategoryTags -ne $null) {
+            $content | add-member -Force -Name "CategoryTags" $fallbackCategoryTags -MemberType NoteProperty
```

Comment:
For reference - you shouldn't set "-Force" here, since then the plugin couldn't override this by itself. Will fix it myself, thanks!